nexus/tests/integration_tests/unauthorized.rs

```diff
@@ -288,6 +337,26 @@ async fn verify_endpoint(
         Visibility::Protected => StatusCode::NOT_FOUND,
     };
 
+    // For routes with an id param, replace the id param with the setup response if present.
+    let uri = if endpoint.url.contains("{id}") {
+        match setup_response {
+            Some(response) => endpoint.url.replace(
+                "{id}",
+                response
+                    .parsed_body::<IdentityMetadata>()
+                    .unwrap()
+                    .id
+                    .to_string()
+                    .as_str(),
+            ),
+            None => endpoint
+                .url
+                .replace("{id}", "00000000-0000-0000-0000-000000000000"),
```

Comment:
In this case there was no entry in `setup_requests` for some given endpoint that included `{id}`. This is true in the case of unimplemented endpoints like `images` and `snapshots` right now.